src/components/Notifications/EventLog/usePrimaryToolbarFilterConfigWrapper.tsx

```diff
@@ -263,17 +263,11 @@ export const usePrimaryToolbarFilterConfigWrapper = (
         // Only add applications to Query Params under 2 conditions
         //     1. Bundle has children
         //     2. Every application under the Bundle is not selected
-        if (
-          chipValues &&
+
+        chipValues &&
           bundle.children?.some(
             (application) => !chipValues.includes(application.name)
-          )
-        ) {
-          chipValues.forEach((chipValue) => {
-            const applicationQueryParam = `${bundle.name}.${chipValue}`;
-            currApplicationFilters.push(applicationQueryParam);
```

Comment:
the changes look a bit weird, as you removed the forEach, only two conditions producing unused boolean seem to be remaining there.

Reply:
the customFilter is already iterating the chipValues above, removing this stops from re-iterating and adding this extra chip thats not needed, this is how I am interpreting it at least..

Reply:
of course, but 
```
chipValues &&
          bundle.children?.some(
            (application) => !chipValues.includes(application.name)
          );
```
that remained in the code has no effect, we can get rid of it

Reply:
also after removing this line
`currApplicationFilters.push(applicationQueryParam);`
filter values for services are not reflected in the URL anymore, which we probably want to keep